blablas/ep198/index.md

```diff
@@ -0,0 +1,61 @@
+---
+date: 2024-11-17
+duration: "2:32:14"
+title: "AI for good"
+tags: ["AI"]
+category: "ai"
+youtube: https://www.youtube.com/watch?v=xprf2CY62OA
+published: true
+featured: false
+---
+
+
+هاد حلقة فيها نقاش مهم لي كنساوه وسط ترندز والهايب ديال التكنولوجيا الجديدة🤔.
+ولينا كنشوفو بزاف ديال بروجيات AI وفينما ضرتي اتلقى Company X Adopted AI. ولكن واش خاصنا نديرو داكا غي حيت هادا العصر ديالو أو لا خاص نفكرو أنه تكنولوجيا جات باش تحل مشاكل كينا فالواقع وتسهل حياتنا أو ترفع من جودتها ومنديروهاش غي باش منبانوش "قدام"🙄.
+
+## Guests
+
+- [Taha Bouhsine](https://twitter.com/Tahabsn)
+- [Ayoub Tarik ](https://www.linkedin.com/in/ayoubtariq01)
+- [Sara El-Ateif](https://www.linkedin.com/in/sara-el-ateif/)
+
+## Notes
+
+00:00:00 - Intro and welcoming guests
+
+00:02:26 - How would you define "AI for Good" and why is it important today? Can you share your journey or experience with AI and social good projects? 
+
+00:15:02 - What are some notable examples of AI addressing global challenges, such as healthcare, climate change, or education, and have you worked on any projects where AI made a tangible positive impact?
+
+00:40:00 - What ethical challenges do you encounter when implementing AI solutions for social good?
+
+01:06:46 - How can organizations ensure that AI for good initiatives is inclusive and equitable?
+
+01:26:12 - How can governments, tech companies, and communities collaborate effectively on AI initiatives? What role does open-source software play in promoting AI for good?
+
+02:06:00 - Do you think AI can be used to protect and promote human rights worldwide? Are there any ethical boundaries that AI for good initiatives should never cross?
+
+02:17:40 - What role does education play in preparing future generations to develop and manage ethical AI systems
+
+02:25:00 - Giveaway
+
+02:32:14 - Wrap up & Bye bye
```

Comment:
Can you recheck this note, the timestamp is equal to episode duration, clicking on the timestamp may triiger error

Reply:
Fixek that and english description in another commit, you can check it out